pymatgen/io/vasp/sets.py

```diff
@@ -1125,6 +1125,90 @@ def kpoints(self):
         return kpt
 
 
+class MVLGBVaspInputSet(MPRelaxSet):
+    """
+    Class for writing a vasp input file for grain boundary, slab or bulk.
+    Args:
+        structure(Structure): provide the structure
+        k_product(int): kpts[0][0]*a.
+        default to 40
+        bulk(bool): Set to True if the structure is bulk supercell. Default is False.
+        dope(bool): Set to True if your structure is doped structure. Default is False.
+        slab(bool): Set to True if your structure is slab. Default is False.
+        manual_kpoints(list): e.g. manual_kpoints=[4,4,1]
```

Comment:
What cannot be already achieved by user_kpoint_settings that we need manual_kpoints??